Cargo.toml

```diff
@@ -116,7 +116,7 @@ kvproto = { git = "https://github.com/pingcap/kvproto.git" }
 tikv_alloc = { path = "components/tikv_alloc", default-features = false }
 tikv_util = { path = "components/tikv_util" }
 log_wrappers = { path = "components/log_wrappers" }
-engine = { path = "components/engine" }
+engine = { path = "components/engine", default-features = false }
```

Comment:
why disable default for engine here?

Reply:
Because sometimes I do a terrible job with my `git add` globs. :(